packages/webcert/src/feature/certificate/Buttons/ForwardCertificateButton.tsx

```diff
@@ -53,7 +55,7 @@ const ForwardCertificateButton: React.FC<Props> = ({
       name={name}
       startIcon={<FontAwesomeIcon size="lg" icon={faReply} />}
       modalTitle="Markera som vidarebefordrad?"
-      onConfirm={() => dispatch(forwardCertificate(true))}
+      onConfirm={() => dispatch(forwardCertificate({ certificateId: certificateId, version: version, forward: true }))}
```

Comment:
Looks like `forwardCertificate` needs to updated to accept this new interface.

![image](https://user-images.githubusercontent.com/28404/177775993-36c8c85f-37ec-46e0-961b-bf9ab58f0f1e.png)

Reply:
You're right! I had forgotten to commit two files. Now they're up :)